charts/objecten/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.9.0 (2025-08-21)
+##### Upgraded 
+- Redis Bitnami Helm subchart to version `22.0.1`
+- Common Bitnami Helm subchart to version `2.31.4`
+##### Changed
+- Redis: Migrated from Bitnami to official Redis container image (`8.0.3`) [pinned] 
```

Comment:
Same question as in the OF PR 😄 

https://github.com/maykinmedia/charts/pull/268#discussion_r2290272339